src/amo/components/Home/styles.scss

```diff
@@ -1,5 +1,10 @@
 @import '~amo/css/styles';
 
+*[dir='rtl'] {
+  background-position: right;
+  unicode-bidi: embed;
+}
```

Comment:
hi @hritvi,

I think you're on the right track here :) but we *only* want to do this in scope of the home icon here. So something like ~:

```css
.Home-SubjectShelf-list-item .Icon
     background-position: left; 
     …other styles


     [dir='rtl'] & {
        background-position: right; // this will override the above style only when it's rtl

         // you’ll probably need to add here the large breakpoint too to change it
         // back to center again 
      }
   ….
```

see before:
![Alt text](https://monosnap.com/image/8U2rOdZIZQUwW0biJ9n5vkOdB6nhi8.png)

after (with icons more to the right):
![Alt text](https://monosnap.com/image/EiiodVXI1RV0q59vTYGM5xljTGwiR7.png)

does that make sense?

Reply:
![screenshot from 2018-09-13 12-40-57](https://user-images.githubusercontent.com/32164618/45472745-bc59c080-b752-11e8-8f59-0c659e53c144.png)
Does this works?